psutil/tests/test_contracts.py

```diff
@@ -282,6 +284,9 @@ def test_fetch_all(self):
             'send_signal', 'suspend', 'resume', 'terminate', 'kill', 'wait',
             'as_dict', 'parent', 'children', 'memory_info_ex', 'oneshot',
         ])
+        if AIX:
+            # "<exiting>" processes really don't have names
+            excluded_names.add('name')
```

Comment:
I wouldn't do this. Instead I would change the `name` method (defined later) like this:

```
    def name(self, ret, proc):
        self.assertIsInstance(ret, str)
        if not AIX:
            assert ret
        # ...else on AIX "<exiting>" processes (zombies) don't have names
```